connectors/db/vector.py

```diff
@@ -256,7 +257,7 @@ def initialize(self):
         self.search_providers = [
             MMRSearchProvider(self.store),
             SimilaritySearchProvider(self.store),
-            HybridSearchProvider(self.store),
+            HybridSearchProvider(self.store) if cfg.ENABLE_HYBRID_SEARCH else None,
```

Comment:
You won't want to append `None` because you'll end up calling `None.search` later on